modules/402-ingress-nginx/hooks/requirements.go

```diff
@@ -52,5 +56,14 @@ func init() {
 		return true, nil
 	}
 
-	requirements.Register("ingressNginx", f)
+	disruptionCheckFunc := func() (bool, string) {
+		reason := ""
+		if hasDisruptionVersionUpdate {
+			reason = "Default IngressNginxController version 0.33 will be automatically changed to 1.1. This action will restart all controllers with non-specified version"
+		}
+		return hasDisruptionVersionUpdate, reason
+	}
+
+	requirements.Register("ingressNginx", checkRequirementFunc)
+	requirements.RegisterDisruptionFunc("disruption:ingressNginx", disruptionCheckFunc)
```

Comment:
Why do we need to specify `disruption` prefix here explicitly?

Reply:
We don't. It would be added automatically if not set. But I decided to spread names here just for eye-simplicity

Reply:
When I see "ingressNginx" I cannot understand if there is an implicit prefix or no prefix at all. I suggest adding prefixes explicitly to make it less confusing. Probably, we should introduce "check" prefix for it.

Reply:
Oh, I don't mean to add them here, only in requirements.json. Method names `RegisterCheck` and `RegisterDisruption` are clean enough.